src/protocols/secure_channel/CASESession.cpp

```diff
@@ -987,11 +994,13 @@ CHIP_ERROR CASESession::ConstructSaltSigmaR2(const System::PacketBufferHandle &
 CHIP_ERROR CASESession::ConstructSaltSigmaR3(const uint8_t * ipk, size_t ipkLen, System::PacketBufferHandle & salt)
 {
     uint8_t md[kSHA256_Hash_Length];
+    memset(salt->Start(), 0, salt->DataLength());
```

Comment:
Add TODO about NEVER using a fixed salt. This needs to be randomly generated.

Reply:
The salt is actually computed using the commissioning hash and IPK. The change above is just resetting buffer to 0 before filling it in.